src/types/publication_types/ticket.types.ts

```diff
@@ -24,7 +24,7 @@ export interface TicketCollection {
 }
 
 export type TicketType = 'DoiRequest' | 'GeneralSupportCase' | 'PublishingRequest';
-export type TicketStatus = 'New' | 'Pending' | 'Closed' | 'Completed';
+export type TicketStatus = 'New' | 'Pending' | 'Closed' | 'Completed' | 'Not Applicable';
```

Comment:
Burde ikke denne bruke `PascalCase` som de fleste andre enums vi bruker i dag (feks `TicketType` rett over)?
Sparer oss fort en skitten migreringssjobb senere om vi tar det nå
```suggestion
export type TicketStatus = 'New' | 'Pending' | 'Closed' | 'Completed' | 'NotApplicable';
```